### ISSUE TITLE
WebAIM - Creating Accessible Forms

### ISSUE THREAD

**Issue by fititnt** [webiwg/acessibilidade-web-pt#12]
**Rascunho de trabalho**
- http://acessibilidade.pt.webiwg.org/webaim/tecnicas/formularios/

**Link do original**: 
- http://webaim.org/techniques/forms/
- http://webaim.org/techniques/forms/controls
- http://webaim.org/techniques/forms/advanced

---

**Descrição**: _Forms are used for many types of interactions on the web. When we talk about the accessibility of forms, we are usually referring to their accessibility to people who use screen readers or keyboards. People with other types of disabilities are generally less affected by faulty forms. It should be noted, however, that everyone benefits from a well-organized, highly usable form, especially those with cognitive disabilities._
**Existe traduções em português**: não
## Ações
- [x] Definir pertinência
- [x] Descobrir se é necessário autorização prévia para traduzir
- [x] Caso seja necessário, solicitar autorização para traduzir
- [x] Adicionar versão original ao repositório
- [x] Definir uma ou mais pessoas para traduzir original
- [x] Completar tradução para português
- [ ] Revisar tradução feita

**Comment by fititnt** [comment]
@webiwg/colaboradores Eu assumo como tradutor principal desse tópico aqui.

Fico no aguardo de alguém pra entrar em contato com a equipe do WebAIM pra pedir autorização pra vermos se vamos adiante ou não na tradução.

O link para versão em espanhol deles está fora do ar, e não há link para versão em português.

**Comment by Dkmister** [comment]
Irei enviar os emails ASP.

**Comment by Dkmister** [comment]
Enviado com sucesso.

**Comment by fititnt** [comment]
Acabei de adicionar referência para tradução de duas sub-páginas que estão inclusas nessa principal.

O ponto que estou vendo aqui é que esse documento aqui não vai ser muito trivial para preparar ele para tradução, pois ele não envolve só texto, envolve também código html e links entre as páginas.

Tende a valer mais a pena, prevendo que a gente pode traduzir mais na frente vários artigos da WebAIM, já fazer até CSS e afins pronto para esse site

**Comment by fititnt** [comment]
@Dkmister esse aqui não tinha sido autorizado? Se sim, _you know the drill_

**Comment by Dkmister** [comment]
Recebi o email da webaim com a seguinte mensagem:
Vilmar -

You are very welcome to translate any of our resources. We do require that:
- The translation be high quality and not automated. Translation
  quality is verified (we have fluent Spanish speakers on staff)
- A link be provided to the original article on our site.
- That "Copyright - WebAIM" be presented at the top or bottom of the
  translation.
- That links remain in-tact (they are not re-written or hijacked).
- That styles, images, and other media be copied to and provided on
  your own site rather than hot-linked from our site. This ensures that
  they will work even if the originals change. In the case of movies,
  etc., you can link to our site.
- That you send me the URL of the translations when they are completed
  so I can link to them from the original.

Thanks,

Jared Smith
WebAIM.org

**Comment by fititnt** [comment]
Nota: vale a pena também traduzir o artigo http://webaim.org/techniques/css/invisiblecontent/. Ele é muito importante, e não é tão grande assim.

Ele é referenciado por esta tradução aqui.

**Comment by fititnt** [comment]
Tradução confluída. Agora é alguém, diferente de mim, disponível pra revisar.

Posso ter feito erros na tradução, porém é mais provável que tenha erros de português, em especial na concordância.

Eu estou impressionado com a qualidade desse texto. Da pra perceber que ele foi revisado por múltiplas pessoas da forma como ele foi escrito.

**Comment by fititnt** [comment]
O Daniel confirmou que vai revisar esse texto aqui.

**Comment by fititnt** [comment]
Publicado como versão para uso geral, removido situação de candidata a lançamento.

Agora é entrar em contato com autor original e reportar tradução, além de divulgar a respeito. Parabens a todos os envolvidos no processo, em especial o @ddtaoli, que foi sua primeira contribuição aqui na @WebIWG!

**Comment by Dkmister** [comment]
Posso fechar o issue, @fititnt ?

**Comment by fititnt** [comment]
Ainda não. Na pratica, temos que informar para a WebAIM que a tradução foi feita, e que podem divulgar. Creio que feito isso, você pode vir aqui e fechar. 

Você manda email disso? Se sim, troca o ultimo assignee como você removendo eu, cola aqui o texto do e-mail enviado e fechamos.

**Comment by Dkmister** [comment]
@fititnt posso mandar o email então?

**Comment by fititnt** [comment]
SIm

**Comment by Dkmister** [comment]
Enviei e-mail para eles

> Jared,
> 
> I'm answering you about the email that I've sent on behalf of the group webIWG http://www.webiwg.org/en/. We've finished the translation of some texts, you can see in this link:http://acessibilidade.pt.webiwg.org/webaim/tecnicas/formularios/ of your followings texts:
> 
> ```
> http://webaim.org/techniques/forms/
> http://webaim.org/techniques/forms/controls
> http://webaim.org/techniques/forms/advanced
> ```
> 
> I look forward to your reply.
> 
> []s
> 
> Vilmar Dorneles Aprato Neto, member of WebIWG
